site/proxy.md

```diff
@@ -104,6 +104,12 @@ Multiple IP addresses and networks can be supplied in the `WEAVE_CIDR`
 variable by space-separating them, as in
 `WEAVE_CIDR="10.2.1.1/24 10.2.2.1/24"`.
 
+The docker IP will still be returned by `docker inspect`. If you want
```

Comment:
This is assuming regular `bridge` networking. Do we need to qualify what happens when you run a container with `--net=none` or `host` ?